dataland-frontend/tests/e2e/specs/prepopulation/AwaitPrepopulation.ts

```diff
@@ -40,7 +40,9 @@ describeIf(
         },
       },
       () => {
-        cy.wait(5000)
+        const delayToWaitForPrepopulationSoThatNotAllRetriesAreWastedInstantly = 5000;
+        // eslint-disable-next-line cypress/no-unnecessary-waiting
+        cy.wait(delayToWaitForPrepopulationSoThatNotAllRetriesAreWastedInstantly)
           .then(() => getKeycloakToken(reader_name, reader_pw))
           .then({ timeout: 120000 }, async (token) => {
```

Comment:
i know its not your code, but is the 120 seconds not saved in a cypress env? (long wait time?)

Reply:
It is not. But if I remember correctly, the long wait time is required(especially for eurodat-backed frameworks.